executor/main.go

```diff
@@ -133,7 +134,7 @@ func runHttpServer(lis net.Listener, logger logr.Logger, predictor *v1.Predictor
 func runGrpcServer(lis net.Listener, logger logr.Logger, predictor *v1.PredictorSpec, client seldonclient.SeldonApiClient, port int, serverUrl *url.URL, namespace string, protocol string, deploymentName string, annotations map[string]string) {
 	grpcServer, err := grpc.CreateGrpcServer(predictor, deploymentName, annotations, logger)
 	if err != nil {
-		log.Fatalf("Failed to create grpc server: %v", err)
+		log.Fatalf("Failed to create gRPC server: %v", err)
```

Comment:
I just realised we should be more consistent in spelling this across docs. 
I think in we often refer to it as just `GRPC` (all capitalised in docs).

Reply:
I think that the right form is `gRPC` according to their site and logo: https://grpc.io/

Reply:
Yup, my point was that we should change in other places: `GRPC` -> `gRPC`